### PR TITLE
created organization edit page

### PR DESCRIPTION
**Pull request by halalak**

### CODE REVIEW

File: client/src/components/edit_organization/MediaInfo.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { AppBar, IconButton } from 'material-ui'
+import './editOrg.css'
+import { style } from '../../widgets/styles'
+import Photo from 'material-ui/svg-icons/image/add-a-photo'
+import { StyledPaper } from '../../widgets/StyledPaper'
+import { StyledTextField } from '../../widgets/StyledTextField'
+import { orange400 } from 'material-ui/styles/colors'
```
Comment (djarredon) [inline client/src/components/edit_organization/MediaInfo.js]: orange400 isn't being used anywhere and is causing linting errors.

File: client/src/components/edit_organization/About.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { AppBar, IconButton } from 'material-ui'
+import { StyledTextField } from '../../widgets/StyledTextField'
+import './editOrg.css'
+import { style } from '../../widgets/styles'
+import { StyledPaper } from '../../widgets/StyledPaper'
+import Company from 'material-ui/svg-icons/communication/business'
+import { orange400 } from 'material-ui/styles/colors'
```
Comment (djarredon) [inline client/src/components/edit_organization/About.js]: orange400 isn't being used and is causing linting errors.

File: client/src/widgets/StyledTextField.js [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import TextField from 'material-ui/TextField'
 import { style } from './styles'
+import { orange400 } from 'material-ui/styles/colors'
 
 export const StyledTextField = props => {
-  const { text, multiLine, disabled } = props
+  const { text, multiLine, disabled, textStyle, org } = props
```
Comment (djarredon) [inline client/src/widgets/StyledTextField.js]: textStyle isn't being used and is causing linting errors.

File: client/src/widgets/StyledTextField.js [hunk 4]
```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import TextField from 'material-ui/TextField'
 import { style } from './styles'
+import { orange400 } from 'material-ui/styles/colors'
```
Comment (djarredon) [inline client/src/widgets/StyledTextField.js]: orange400 isn't being used and is causing linting errors.

**Review by djarredon — CHANGES_REQUESTED** [review]
I forget what had been said about having a save button, but I think a save button should be added either into the left hand column or under the forms, unless we plan on saving user input intermittently, in which case we'll need to work on displaying that to the user. The linting errors need to be fixed and then I'll approve.

**Review by mballeza — CHANGES_REQUESTED** [review]
-Run "yarn eslint src/" to see the list of lint errors.
-The "Information" and "Media" StyledPaper components have the "clickable" cursor when hovered over.
-Do we want the website link and video url to not be multiline like the description?

**Review by mballeza — COMMENTED** [review]
Still has several linting errors. However, you can ignore the "Component should be written as a pure function" warning since we're probably going to add more to the class later.

**Review by djarredon — DISMISSED** [review]

File: client/src/constants/routing.js [hunk 5]
```diff
@@ -41,6 +42,11 @@ export const routePathToComponent = [
     name: 'EditProfile',
     path: '/edit',
     component: ConnectedEditProfile
+  },
+  {
+    name: 'EditOrg',
+    path: '/edit_organization',
```
Comment (iandusenbury) [inline client/src/constants/routing.js]: We'll need to come up with a convention for this. We should probably have it be something like `/{org_id}/edit`

File: client/src/widgets/StyledTextField.js [hunk 6]
```diff
@@ -3,12 +3,14 @@ import TextField from 'material-ui/TextField'
 import { style } from './styles'
 
 export const StyledTextField = props => {
-  const { text, multiLine, disabled } = props
+  const { text, multiLine, disabled, org } = props
   const isMultiLine = !!multiLine
```
Comment (iandusenbury) [inline client/src/widgets/StyledTextField.js]: isn't this the same as writing just `multiLine`?

File: client/src/components/edit/Experience.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 import { style } from '../../widgets/styles'
 import Location from 'material-ui/svg-icons/communication/location-on'
 import { StyledPaper } from '../../widgets/StyledPaper'
 import { AddExperience } from './AddExperience'
+import './experience.css'
 
-export const Experience = props => (
+export const Experience = () => (
   <StyledPaper>
     <div className="EditBoxGrid">
```
Comment (iandusenbury) [inline client/src/components/edit/Experience.js]: kind of nitpicky but class names should be camelCase

File: client/src/constants/routing.js (hunk 5 above)
Comment (halalak) [reply client/src/constants/routing.js]: that's a good point, we'll need to do the same thing for the professional edit page as well then

File: client/src/widgets/StyledTextField.js (hunk 6 above)
Comment (halalak) [reply client/src/widgets/StyledTextField.js]: yes....I don't remember why I thought it was needed at the time, but I'm just going to remove that and use the passed in multiLine prop

**Review by mballeza — CHANGES_REQUESTED** [review]
Looks good! Hovering over the "Information" and "Media" headers makes the mouse change to a hand pointer. Other than that this needs an update with the current dev branch and should be ready to go.

**Comment by djarredon** [comment]
I'm unable to change the video on organizations. I input a new url, but the video stays the same.

**Comment by halalak** [comment]
@djarredon after you hit save it doesn't change the video link?

**Comment by halalak** [comment]
I will look into that issue and let you know when its fixed

**Comment by mballeza** [comment]
@djarredon @halalak 
The issue was that the places reducer used in the submit handler had empty values for the placesResults. I've made an update to the autocomplete component that initializes the state to contain the placesResult from the initialized address value instead of being empty. This should allow you to update the video url.

**Comment by djarredon** [comment]
So I'm able to update the video and description now, but only after changing the address. If I leave the address alone then none of the changes I try to make are saved, but if I edit the address then I can edit the rest of the fields.

**Comment by mballeza** [comment]
@djarredon I've been able to change either the video url or the description without needing to change the address. Let me know if that keeps happening.

**Comment by djarredon** [comment]
I get the following error when navigating to the organization edit page from the admin page. The page loads just fine on a refresh.
edit organization error

**Review by djarredon — APPROVED** [review]

**Review by mballeza — APPROVED** [review]

**Pull request merged**